public/app/features/dashboard/components/DashNav/DashNavTimeControls.tsx

```diff
@@ -88,6 +88,7 @@ export class DashNavTimeControls extends Component<Props> {
     const timePickerValue = getTimeSrv().timeRange();
     const timeZone = dashboard.getTimezone();
     const styles = getStyles();
+    const hideRefreshPicker = !dashboard.panelInEdit?.isEditing;
```

Comment:
Hmm, can we reverse the naming of this variable? Shouldn't it be showRefreshPicker?

Reply:
Thanks for the suggestion :D, I also did a small fix here as well,  this component also had the `refresh/sync` button that needs to be always present https://github.com/grafana/grafana/pull/34128/commits/02f4f35075001434564b9a2244a10706936f7b8c#diff-aac52f3778332268268f492b861c176a728fa4e7f05a4a5574228ec00920482aR91

Reply:
Looks great 👍